sled-agent/src/http_entrypoints.rs

```diff
@@ -315,10 +315,10 @@ async fn vpc_firewall_rules_put(
     body: TypedBody<VpcFirewallRulesEnsureBody>,
 ) -> Result<HttpResponseUpdatedNoContent, HttpError> {
     let sa = rqctx.context();
-    let vpc_id = path_params.into_inner().vpc_id;
+    let _vpc_id = path_params.into_inner().vpc_id;
```

Comment:
Just ignoring this seems a little suspicious. Can we maybe eliminate the path parameter or check it against something?

Reply:
Yea, it wasn't used before either. I hesitated to change as part o this because it does change the shape of the API a bit (though this is just sled-agent's API which AFAIK only nexus calls). I'll open a follow-up issue for possibly changing that.

Reply:
https://github.com/oxidecomputer/omicron/issues/3355